### PR TITLE
doc: add some undocumented options to rbd-nbd

### PR DESCRIPTION
**Pull request by wangzhengyong**
Signed-off-by: wangzhengyong@cmss.chinamobile.com

### CODE REVIEW

**Comment by wangzhengyong** [comment]
@liupan1111 done

**Comment by liupan1111** [comment]
@trociny please do The merge if you feel ok.

**Comment by trociny** [comment]
@wangzhengyong You also need to update Synopsis

**Comment by wangzhengyong** [comment]
@trociny sorry, i forgot it

File: doc/man/8/rbd-nbd.rst [hunk 1]
```diff
@@ -28,11 +28,23 @@ Options
    Use *ceph.conf* configuration file instead of the default
    ``/etc/ceph/ceph.conf`` to determine monitor addresses during startup.
 
+.. option:: --read-only
+
+   Map readonly.
```
Comment (dillaman) [inline doc/man/8/rbd-nbd.rst]: Nit: "read-only"

File: doc/man/8/rbd-nbd.rst [hunk 2]
```diff
@@ -28,11 +28,23 @@ Options
...
    Override the parameter of NBD kernel module when modprobe, used to
    limit the count of nbd device.
 
+.. option:: --max_part *limit*
+
+    Override for module param nbds_max.
+
+.. option:: --exclusive
+
+   Forbid other clients write.
```
Comment (dillaman) [inline doc/man/8/rbd-nbd.rst]: Nit: "Forbid writes by other clients"

Comment (trociny) [inline doc/man/8/rbd-nbd.rst]: @dillaman Oh, sorry, I just merged it and did not noticed your comments.

Comment (trociny) [inline doc/man/8/rbd-nbd.rst]: @dillaman May be then also fix this in ` --help` output?

**Comment by liupan1111** [comment]
@trociny I've create a PR to fix the nits.

**Pull request merged**